sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/Statsbeat.cs

```diff
@@ -63,11 +109,20 @@ private static string GetOS()
 
         internal static void InitializeAttachStatsbeat(string connectionString)
         {
-            if (s_attachStatsBeatMeterProvider == null)
+            // check whether it is disabled or already initialized.
+            if (s_isEnabled && s_attachStatsBeatMeterProvider == null)
```

Comment:
The fact that this _could_ be initialized twice seems wrong. 
Will each exporter have a unique StatsBeat instance, or will the entire SDK have a single StatsBeat instance?

Consider that some customers will send telemetry to different IKeys.

What happens if a root customer has their own customers. The root customer may have an ikey, and the nested customers may have a unique ikey. I don't think we can assume that each customer will be in the same region.

Reply:
There will be only one instance of StatsBeat.

```Consider that some customers will send telemetry to different IKeys.

What happens if a root customer has their own customers. The root customer may have an ikey, and the nested customers may have a unique ikey. I don't think we can assume that each customer will be in the same region.```

This ^ should not be an issue as far as all those ikeys are in EU or NonEU regions.

We will not add Statsbeat if customer tries to to use connection string from multiple regions i.e. one from EU and other from NonEU. I will fix this part in follow up PR.